charts/shinyproxy/Chart.yaml

```diff
@@ -15,15 +15,15 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.0.0
+version: 3.0.0
```

Comment:
are you sure about a major jump of version ?

Reply:
no, thx